backpack/__init__.py

```diff
@@ -142,7 +142,11 @@ def hook_store_io(module, input, output):
     if disable.should_store_io() and torch.is_grad_enabled():
         for i in range(len(input)):
             setattr(module, "input{}".format(i), input[i])
-        module.output = output
+        if isinstance(output, tuple):
+            # is true for RNN,GRU,LSTM which return tuple (output, ...)
+            module.output = output[0]
+        else:
+            module.output = output
```

Comment:
Cool. I know the docstring of this function has potential for improvement. Could you update it as well?

Reply:
Sure. Is done now.